packages/mermaid/src/accessibility.spec.ts

```diff
@@ -158,55 +127,71 @@ describe('accessibility', () => {
+            const svgSelection = select<SVGSVGElement, never>('svg');
+            addSVGa11yTitleDescription(svgSelection, a11yTitle, a11yDesc, givenId);
+            const svgNode = ensureNodeFromSelector('svg');
+            expect(svgNode.getAttribute('aria-describedby')).toBe(`chart-desc-${givenId}`);
+          });
+
+          jsdomIt(
+            'should insert desc tag as the 2nd child with the text set to accDescription given',
+            () => {
+              const svgSelection = select<SVGSVGElement, never>('svg');
+              addSVGa11yTitleDescription(svgSelection, a11yTitle, a11yDesc, givenId);
+              const svgNode = ensureNodeFromSelector('svg');
+              const descNode = ensureNodeFromSelector('desc', svgNode);
+              expect(descNode?.innerHTML).toBe(a11yDesc);
+            }
+          );
         });
 
         describe('without a11y description', () => {
           const a11yDesc = undefined;
 
-          it('should not set aria-labelledby', () => {
-            const svgAttrSpy = vi.spyOn(fauxSvgNode, 'attr').mockReturnValue(fauxSvgNode);
-            addSVGa11yTitleDescription(fauxSvgNode, a11yTitle, a11yDesc, givenId);
-            expect(svgAttrSpy).not.toHaveBeenCalledWith('aria-labelledby', expect.anything());
+          jsdomIt('should not set aria-labelledby', () => {
+            const svgSelection = select<SVGSVGElement, never>('svg');
+            addSVGa11yTitleDescription(svgSelection, a11yTitle, a11yDesc, givenId);
+            const svgNode = ensureNodeFromSelector('svg');
+            expect(svgNode.getAttribute('aria-labelledby')).toBeNull();
           });
 
-          it('should not set aria-describedby', () => {
-            const svgAttrSpy = vi.spyOn(fauxSvgNode, 'attr').mockReturnValue(fauxSvgNode);
-            addSVGa11yTitleDescription(fauxSvgNode, a11yTitle, a11yDesc, givenId);
-            expect(svgAttrSpy).not.toHaveBeenCalledWith('aria-describedby', expect.anything());
+          jsdomIt('should not set aria-describedby', () => {
+            const svgSelection = select<SVGSVGElement, never>('svg');
+            addSVGa11yTitleDescription(svgSelection, a11yTitle, a11yDesc, givenId);
+            const svgNode = ensureNodeFromSelector('svg');
+            expect(svgNode.getAttribute('aria-describedby')).toBeNull();
           });
 
-          it('should not insert title tag', () => {
-            const fauxTitle: MockedD3 = new MockedD3();
-            const svginsertpy = vi.spyOn(fauxSvgNode, 'insert').mockReturnValue(fauxTitle);
-            addSVGa11yTitleDescription(fauxSvgNode, a11yTitle, a11yDesc, givenId);
-            expect(svginsertpy).not.toHaveBeenCalledWith('title', ':first-child');
+          jsdomIt('should not insert title tag', () => {
+            const svgSelection = select<SVGSVGElement, never>('svg');
+            addSVGa11yTitleDescription(svgSelection, a11yTitle, a11yDesc, givenId);
```

Comment:
Would it be possible to provide the svg as a lazily evaluated fixture in test, so we can do something like this?

```suggestion
          jsdomIt('should not insert title tag', ({ svg }) => {
            addSVGa11yTitleDescription(svg, a11yTitle, a11yDesc, givenId);
```

Would make the DX easier. Same with `body`